### ISSUE TITLE
SwaggerUI version compatibility

### ISSUE THREAD

**Issue by cer** [eventuate-clients/eventuate-util#7]
https://github.com/eventuate-clients/eventuate-util/blob/1d24c6365333f8b85519a6206429a4d5e2c335fe/gradle.properties#L5

@dartartem why was this particular version chosen?

**Comment by dartartem** [comment]
It is the same version that used in other services

**Comment by cer** [comment]
Please provide more information. Why are the other services using a particular version?

**Comment by dartartem** [comment]
@cer Chris,

services use this:

    compile "io.springfox:springfox-swagger2:2.9.2"
    compile "io.springfox:springfox-swagger-ui:2.9.2"

it is the latest version of springfox-swagger.

I found that swagger-ui version that used by spring-fox is 3.17.1 

I did by executing this: "JSON.stringify(versions)" in console according: https://swagger.io/docs/open-source-tools/swagger-ui/usage/version-detection/

But as I know api-gateway cannot use springfox swagger integration, that is why we use original swagger ui. You said that ui is inconsistent between apigateway and other services in eventuate-tram-examples-customers-and-orders. So I solved it by using the same version of ui.
